stanford_profile.post_update.php

```diff
@@ -46,3 +49,56 @@ function stanford_profile_post_update_8014() {
   ]);
 
 }
+
+/**
+ * Create blocks for sites with custom themes that were added to stanford_basic.
+ */
+function stanford_profile_post_update_8015() {
```

Comment:
shouldn't this be in the `stanford_profile_helper`?

Reply:
also.... NOW ya make this, after i manually did this for CS. haha

Reply:
Block placement is more of a profile responsibility imho. Why would this be better off in the helper?

Reply:
we'd just have to copy it over to the soe_profile. not much effort but its there

Reply:
Soe profile already has the blocks exported and tracked as config. They are guud.

Reply:
https://github.com/SU-SOE/soe_profile/pull/64/files?file-filters%5B%5D=.yml#diff-5de83d2074b91ef69c0fd270b795b313

Reply:
meh. so it doesn't matter that much then. Good enough to me.